gitlab_runner/datadog_checks/gitlab_runner/config_models/instance.py

```diff
@@ -84,9 +85,9 @@ class Config:
     label_to_hostname: Optional[str]
     labels_mapper: Optional[Mapping[str, Any]]
     log_requests: Optional[bool]
-    metrics: Sequence[str]
+    metrics: Optional[Sequence[Union[str, Mapping[str, str]]]]
     min_collection_interval: Optional[float]
-    namespace: str
+    namespace: Optional[str]
```

Comment:
Is it linked to the feature? should we create another PR for this?

Reply:
it was generated by `ddev validate models -s` which I had to run. @FlorianVeaux is it expected?

Reply:
These changes are already included on master with this PR: https://github.com/DataDog/integrations-core/pull/9401/files
If you rebase and fix potential conflicts, this changes should disappear

Reply:
thanks!